Documentation/network/servicemesh/istio.rst

```diff
@@ -36,35 +36,45 @@ You can confirm the result with the following command:
     $ kubectl get configmaps -n kube-system cilium-config -oyaml | grep bpf-lb-sock-hostns
     bpf-lb-sock-hostns-only: "true"
 
+
+To ensure that Cilium does not interfere with other CNI plugins on the node, it is important to set the
+``cni-exclusive`` parameter in the Cilium ConfigMap to ``false``. This can be achieved by using the
+``--set`` flag with the ``cni.exclusive`` Helm value set to ``false``.
+You can confirm the result with the following command:
+
+.. code-block:: shell-session
+
+    $ kubectl get configmaps -n kube-system cilium-config -oyaml | grep cni-exclusive
+    cni-exclusive: "false"
+
 .. _gsg_istio_cnp:
 
 Istio configuration
 ===============================
 
-When you deploy Istio, be aware of:
-
-* The new experimental `ambient mesh <https://istio.io/latest/blog/2022/introducing-ambient-mesh/>`_ 
-  data plane is not supported, as it interferes with the Cilium data plane.
+When you deploy Cilium and Istio together, be aware of:
 
-* mTLS mode ``STRICT`` or ``PERMISSIVE`` (default) are not compatible with Cilium HTTP network policy. 
-  To use an HTTP-based network policy (for example, :ref:`l7_policy`), 
-  you must configure ``mtls.mode=DISABLE`` under Istio's ``PeerAuthentication``.
+* Either Cilium or Istio L7 HTTP policy controls can be used, but you cannot use **both** Cilium and Istio
+  L7 HTTP policy controls at the same time.
+  To use Cilium L7 HTTP policy controls (for example, :ref:`l7_policy`) with Istio, you must disable Istio mTLS by configuring
+  ``mtls.mode=DISABLE`` under Istio's ``PeerAuthentication``. Using Istio's L7 HTTP policy controls instead of Cilium's
+  is out of scope for this guide, refer to the Istio documentation for additional details.
 
-* When using Kubernetes admission webhooks to `inject sidecar proxies <https://istio.io/latest/docs/ops/configuration/mesh/webhook/>`_
+* When using Istio in sidecar mode with Kubernetes admission webhooks to `inject sidecar proxies <https://istio.io/latest/docs/ops/configuration/mesh/webhook/>`_
```

Comment:
@bleggett can you help clarify "When using Istio in sidecar mode with Kubernetes admission webhooks"? From my understanding, this is Istio sidecar mode with [automatic sidecar injection](https://istio.io/latest/docs/setup/additional-setup/sidecar-injection/#automatic-sidecar-injection), correct? If so, can you update accordingly?